code/datums/ai_laws.dm

```diff
@@ -24,170 +56,338 @@
 			return ai_law
 	return null
 
+
+// ********* Getters for law descriptions *********
+
+/datum/ai_laws/proc/get_desc_devil_law_sixsixsix(index)
+	var/ret = null
+	if (index < 1 || index > devil_laws.len)
+		return ret
+	if (devil_laws[index])
+		var/datum/ai_law/law = devil_laws[index]
+		ret = law.description
+	return ret

```

Comment:
You don't need to change this or anything, but for future reference, you could have used . instead of ret and just do . = law.description, removed the last return ret, and changed the first one to just return and wound up with the same result with less typing required.